IoTCloudSDK/IoTCloudSDK/IoTCloudAPI.swift

```diff
@@ -380,6 +382,23 @@ public class IoTCloudAPI: NSObject, NSCoding {
 
         return newIotapi
     }
+    
+    func saveToUserDefault(){
+        NSUserDefaults.standardUserDefaults().setObject(NSKeyedArchiver.archivedDataWithRootObject(self), forKey: "IoTCloudAPI")
+        NSUserDefaults.standardUserDefaults().synchronize()
+    }
+
+    public override func isEqual(object: AnyObject?) -> Bool {
```

Comment:
may be here is internal, instead of public? Otherwise, you need to add doc for it.